src/__stories__/custom-input-story.tsx

```diff
@@ -0,0 +1,269 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import {StorySection} from './helpers';
+import {
+    useTheme,
+    Text2,
+    Text3,
+    TextField,
+    Menu,
+    Touchable,
+    Box,
+    Text1,
+    createUseStyles,
+    Form,
+    Stack,
+} from '..';
+import {DOWN, ENTER, SPACE, UP} from '../utils/key-codes';
+import {cancelEvent} from '../utils/dom';
+import {combineRefs} from '../utils/common';
+
+export default {
+    title: 'Components/Forms/Custom input',
+};
+
+const OPTIONS = [
```

Comment:
Make this a list of countries or fruits or whatever... If all options starts with "Option" it always autocompletes all or nothing.

I would be also interesting for this example to have scroll in the list of suggestions

Reply:
done